compiler/noirc_frontend/src/node_interner.rs

```diff
@@ -1611,6 +1628,7 @@ impl NodeInterner {
             DependencyId::Global(id) => {
                 Cow::Borrowed(self.get_global(id).ident.0.contents.as_ref())
             }
+            _ => Cow::Borrowed(""),
```

Comment:
aren't we multiplying LOC for no real benefit?

Reply:
We've had situations in the past where e.g. new blackbox functions aren't handled properly because we used a catch-all in a match statement which masked when we should be adding handling for this new enum variant. This allows us to catch these errors more easily.

In this case in particular this catch-all is only handling a single enum variant so there's not even a change to LOC.